grr/core/grr_response_core/config/output_plugins.py

```diff
@@ -63,3 +63,22 @@
 
 config_lib.DEFINE_string("Splunk.index", None,
                          "The index assigned to all submitted events.")
+
+# Elasticsearch Output Plugin
+config_lib.DEFINE_string(
+    "Elasticsearch.url", None, "Absolute URL of the Elasticsearch installation,"
+    " e.g. 'https://myelasticsearch.example.com:9200'")
+
+config_lib.DEFINE_string(
+    "Elasticsearch.token", "",
+    "Token used to authenticate with the Elasticsearch cluster.")
+
+config_lib.DEFINE_bool(
+    "Elasticsearch.verify_https", True,
+    "Verify the certificate for HTTPS connections. Setting this to False comes "
+    "with big security risks. Instead, when using self-signed certificates, "
+    "set REQUESTS_CA_BUNDLE environment variable to the path of the cert file. "
+    "See https://requests.readthedocs.io/en/master/user/advanced/.")
+
+config_lib.DEFINE_string("Elasticsearch.index", "grr-flows",
+                         "The index assigned to all submitted events.")
```

Comment:
Missing trailing blank line (as shown by the GitHub UI).

Reply:
VS Code, why you no lint ): fixed in next rev